Cargo.lock

```diff
@@ -2613,7 +2613,7 @@ dependencies = [
  "serde_yaml",
  "static_assertions",
  "thiserror",
- "time 0.2.22",
+ "time 0.2.23",
```

Comment:
so what does it mean here that some dependencies have a version set, and some don't? (cc @sunshowers)

Reply:
(my guess is that dependencies that DON'T have a version set here are using the same version as we use when we directly import them, if this is the case then we should be able to remove the version here.)

Reply:
(interestingly I found https://docs.rs/cargo-lock/6.0.0/cargo_lock/ while searching for the answer)

Reply:
This is the Cargo.lock file, it's not human editable. To answer your question, versions get set when there are multiple versions of the same package and you need to disambiguate by version.

Reply:
Many of the versions in the lock file are set via dependabot prs, which I suspect use cargo-edits, `cargo update` command.

Reply:
Okay so this is only for the kube/rusoto crates which I'm pretty sure are test only code.   So maybe ignored by guppy -- but dependabot?

Reply:
@rexhoffman I think it might be because time is only a transitive dependency, not a direct one...

this is a pretty bad bug though, and maybe we should prioritize doing our own dependabot